apps/web/composables/useHomepageData/useHomepageData.ts

```diff
@@ -1,17 +1,21 @@
-import { ref, watch, computed } from 'vue';
 import { HeroContentProps, SizeKey } from '~/components/ui/HeroCarousel/types';
 import { MediaItemProps } from '~/components/ui/MediaCard/types';
-import { useCategoryTree, useCategoryTemplate } from '~/composables';
-import homepageTemplateData from './homepageTemplateData.json';
+import { useCategoryTemplate } from '~/composables';
```

Comment:
Nuxt scope should recognize the composable. You don't need the import.

Reply:
Done.